plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/ScriptSelectorPanel.java

```diff
@@ -144,11 +146,15 @@ protected Control createDialogArea(Composite parent) {
         });
         final Color fg = patternText.getForeground();//parent.getDisplay().getSystemColor(SWT.COLOR_INFO_FOREGROUND);
         final Color bg = patternText.getBackground();//parent.getDisplay().getSystemColor(SWT.COLOR_INFO_BACKGROUND);
-
+        Composite childComposite = new Composite(composite, SWT.NONE);
+        final GridData layoutData = new GridData(GridData.FILL_HORIZONTAL);
+        GridLayout layout = new GridLayout(2, true);
+        childComposite.setLayout(layout);
+        childComposite.setLayoutData(layoutData);
         composite.setForeground(fg);
         composite.setBackground(bg);
-
-        Button newButton = new Button(composite, SWT.PUSH | SWT.FLAT);
+        Button newButton = new Button(childComposite, SWT.PUSH | SWT.FLAT);
+        Button getProjectScripts = UIUtils.createCheckbox(childComposite, "Get project scripts", false);
```

Comment:
That's kinda weird name because the checkbox itself doesn't _get_ anything.

May it be `Show project scripts` instead?

And also please add postfix `Check` to the checkbox button variable.